src/basic/build.h

```diff
@@ -133,6 +135,8 @@
 #define _IDN_FEATURE_ "-IDN"
 #endif
 
+#define _CGROUP_HIEARCHY_ "default=" STRINGIFY(DEFAULT_HIERARCHY)
```

Comment:
i think it would make a lot of sense to continue with the same syntax we already have, i.e. generate individual boolean feature flags. Specifically, one of these:

``
+DEFAULT_UNIFIED -DEFAULT_HYBRID -DEFAULT_LEGACY
-DEFAULT_UNIFIED +DEFAULT_HYBRID -DEFAULT_LEGACY
-DEFAULT_UNIFIED -DEFAULT_HYBRID +DEFAULT_LEGACY
```

Or eve, call this `CGROUP_DEFAULT_UNIFIED`… and so on, to clarify the cgroup context